contrib/local-environment/oauth2-proxy-keycloak.cfg

```diff
@@ -0,0 +1,20 @@
+http_address="0.0.0.0:4180"
+cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
+email_domains=["example.com"]
+cookie_secure="false"
+upstreams="http://httpbin"
+cookie_domains=[".localtest.me"] # Required so cookie can be read on all subdomains.
+whitelist_domains=[".localtest.me"] # Required to allow redirection back to original requested target.
+
+# keycloak provider
+client_secret="72341b6d-7065-4518-a0e4-50ee15025608"
+client_id="oauth2-proxy"
+redirect_url="http://oauth2-proxy.localtest.me:4180/oauth2/callback"
+
+# in this case oauth2-proxy is going to visit
+# http://keycloak.localtest.me/auth/realms/master/.well-known/openid-configuration for configuration
```

Comment:
```suggestion
# http://keycloak.localtest.me:8080/auth/realms/master/.well-known/openid-configuration for configuration
```

Reply:
switched to port 9080 - 8080 is often occupied by java projects.